types/got/index.d.ts

```diff
@@ -98,18 +104,23 @@ declare namespace got {
         json?: boolean;
     }
 
-    interface TimoutRequestOptions extends http.RequestOptions {
+    interface RequestOptions extends http.RequestOptions {
```

Comment:
Maybe we should name this something "strange" to discourage people from using it? It's only here to work around how we are extending the options...

`_InternalRequestOptions`?

Reply:
Yes, I also think about this. We move this interface from namespace.

Reply:
Perfect 👍